src/css/info.scss

```diff
@@ -39,37 +45,40 @@
       }
     }
 
-    &:before {
-      content: "";
-      position: absolute;
-      left: 0;
-      right: 0;
-      top: 0;
-      bottom: 0;
-      min-width: 856px;
+    .gridImgColumn  {
+      max-width: 381px;
+      height: 1115px;
+      margin-right: auto;
+      margin-left: -20px;
       background-image: url(../images/info-grid.png);
-      background-repeat: no-repeat;
-      background-position: left bottom;
-      background-size: 275px;
-      transform: rotate(-90deg);
-
-      @media (min-width: #{$bp-medium}) {
-        background-size: 381px;
-        background-position: left;
-        background-position-x: 38px;
-        transform: rotate(0deg);
-      }
-    }
-
-    &:after {
-      content: "";
-      position: absolute;
-      left: 0;
-      right: 0;
-      top: 0;
-      bottom: 0;
-      background: rgba($grey-10, 0.96);
-    }
+      background-size: cover;
+      opacity: 0.05;
+
+      @media (max-width: #{$bp-medium - 1}) {
+          background-image: url(../images/info-grid-mobile.png);
+          max-width: 100%;
+          background-size: contain;
+          background-repeat: no-repeat;
+          background-position: bottom;
+          margin-left: 15%;
+          height: 400px;
+          opacity: 0.05;
+      }
+    };
+
+    .privacyColumn  {
+      align-self: center;
+
+      @media (max-width: #{$bp-large}) {
+        padding-left: 45px;
+        h2  {
+          font-size: 45px;
+          line-height: 50px;
+        }
+        p {
+          font-size: 18px;
```

Comment:
`font-size` and `line-height` are already defined in global. Don't need to redefine it in here.